flexget/plugins/api_trakt.py

```diff
@@ -294,6 +294,37 @@ def get_id(type, name=None, year=None, trakt_slug=None, tmdb_id=None, imdb_id=No
 
     # TODO: Fall back to name/year search
 
+    clean_title = normalize_series_name(name)
+    found = None
+    try:
+        results = req_session.get(get_api_url('search'), params={'query': clean_title, 'type': type}).json()
+    except requests.RequestException as e:
+        log.debug('ID Lookup failed on error: %s' % e)
+    if results['type'] is type:
+        for result in results:
+            if clean_title == result[type]['title']:
```

Comment:
Hmm, isn't this going to give us trouble because the symbols are stripped out of clean_title?
On a separate note, I'm actually not so confident in the search by title at all until trakt fixes up results order.